### DISCUSSION TITLE
Will an S40 work without an internet connection?

### DISCUSSION THREAD

**Discussion by ChrisLizon** [PeteRager/lennoxs30#303] [Q&A] [answered]
Hello Everyone, 
I'm nearing the final decision making process for a new Furnace/Heat Pump for my home. The installer is reccomending an S40, which my understanding is a communicating thermostat that will actually talk to the furnace and heat pump and be able to do diagnostics and stuff which sounds awesome.  I think it's probably worth the extra $ over an ecobee or keeping my antique RadioThermostat (cloud is dead, local control lives on!).

From what I see in the documentation this thermostat does offer local control, I am curious if it needs a connection to Lennox's cloud to function or if I can put it on my untrusted IoT vlan with no internet access to ensure that the thermostat never phones home for an upgrade. Is there anything I would loose out on without the thermostat connected to the internet?

Thanks!

**Comment by PeteRager** [comment, marked as answer]
I have my S30 on a VLAN with no internet access and it works fine. That is the preferred configuration. When coupled with Lennox equipment you have complete control of all equipment parameters.

The S40 is really just an updated S30 with some upgrades (Bluetooth sensors)

The only thing you won't have is the ability for Lennox dealer to remotely access your system.  But that can be overcome.

What I would consider doing, is get it setup with a connection to the Lennox cloud and get your Lennox account setup.  Once that is working then block it from the internet.   You can then always unblock it if you need to allow remote access or if you decide using the Lennox app on your phone is useful..  I do this periodically to test the integrations cloud connection.

**Reply by ChrisLizon** [reply]
@PeteRager  thank you very much for taking the time to answer my question. At this rate it looks like I will be installing this component in the near future!
